packages/styled-system/src/config/typography.ts

```diff
@@ -18,6 +18,10 @@ export const typography: Config = {
   whiteSpace: true,
   textDecoration: true,
   textDecor: { property: "textDecoration" },
+  textDecorationColor: t.colors("color"),
+  textDecorationLine: true,
+  textDecorationStyle: true,
```

Comment:
I don't think we need these style props, to be honest, but since we already have `textDecorationColor`, we might want to leave them.

Another way to avoid this work is to use the css var: `textDecorationColor: var(--chakra-colors-green-300)`. I'll write some docs around this soon.